django/db/models/aggregates.py

```diff
@@ -36,6 +36,20 @@ def default_alias(self):
     def get_group_by_cols(self):
         return []
 
+    def get_options(self):
```

Comment:
I'd sort `__repr__` under `__init__` and name this something like `_get_repr_options` and add a docstring to the base class implementation like `Return a dict of extra __init__() options to include in the repr.`
Did you consider trying to put this higher up the inheritance chain than `Aggregate`?